electroncash_plugins/fusion/plugin.py

```diff
@@ -660,10 +676,11 @@ def wallet_can_fuse(wallet) -> bool:
         return can_fuse_from(wallet) and can_fuse_to(wallet)
 
     @staticmethod
-    def is_fuz_coin(wallet, coin) -> Optional[bool]:
+    def is_fuz_coin(wallet, coin, fuz_parents = 0) -> Optional[bool]:
```

Comment:
Apparently python linters hate the extra spaces here in arguments. Go figure.

```suggestion
    def is_fuz_coin(wallet, coin, fuz_parents=0) -> Optional[bool]:
```